README.md

```diff
@@ -210,6 +210,7 @@ Python 3.x, nginx, uwsgi and sqlite and should be sufficient for a few users
 1. Initiate the Python environment
 
         pipenv install --three --dev
+        pipenv shell
```

Comment:
I don't think this is necessary here. If you take a look at your own log in #43, note that `pipenv shell` is never used. It is the other pieces that got you over the hump, I believe. Adding this doesn't actually *hurt* anything, but I would prefer you remove it.

Reply:
i assume it is the bart that make it work because  a frend told me on debian systems make differences when using sudo to user installations i think the environement keeps it in space.